paimon-core/src/main/java/org/apache/paimon/catalog/AbstractCatalog.java

```diff
@@ -296,7 +296,7 @@ private FileStoreTable getDataTable(Identifier identifier) throws TableNotExistE
 
     @VisibleForTesting
     public Path newDatabasePath(String database) {
```

Comment:
You can just override this method?
We can just remove `VisibleForTesting`, and add documentation.

Reply:
Sounds reasonable, I have updated this PR